src/app/utils/view.utils.ts

```diff
@@ -0,0 +1,9 @@
+import { DOMSerializer } from 'prosemirror-model';
+import { EditorState } from 'prosemirror-state';
+
+export function stringifyEditorState(editorState: EditorState): string {
+  const fragment = DOMSerializer.fromSchema(editorState.schema).serializeFragment(editorState.doc.content);
```

Comment:
I thought there must be a better, simpler, way to do this. But after 30 minutes of research it seems like this is accepted approach...